examples/helmAgentAddon/example/policyaddon/main.go

```diff
@@ -0,0 +1,143 @@
+package main
```

Comment:
> examples/helmAgentAddon/example/policyaddon/main.go 

nit: conventionally the folder/file name should be snakecased?

Reply:
done